pkg/topology/run.go

```diff
@@ -359,14 +361,8 @@ func Run(opts TopologyRunOptions, s v1.SystemTemplate) []*pkg.Component {
 				component.Components = append(component.Components, components...)
 				continue
 			}
-			group := pkg.NewComponent(comp)
-			group.Components = append(group.Components, components...)
-			if len(group.Components) > 0 {
-				group.Summary = group.Components.Summarize()
-			}
 
-			group.Status = pkg.ComponentStatus(group.Summary.GetStatus())
```

Comment:
Removing groups because it was creating an unnecessary subgroup. Also, we don't need summary and status since that is calculated at query time